### PR TITLE
Improve warning message in rocksmq

### PR DESCRIPTION
**Pull request by wxyucs**
Signed-off-by: Xiangyu Wang <xiangyu.wang@zilliz.com>

/kind improvement
/cc @fishpenguin

### CODE REVIEW

**Comment by yanliang567** [comment]
@scsven ci failed, maybe retry after rebase would be working

**Comment by scsven** [comment]
@milvus-io/milvus-maintainers

File: internal/util/rocksmq/server/rocksmq/rocksmq_impl.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func (rmq *rocksmq) Close() {
 		if topic != "" {
 			err := rmq.DestroyTopic(topic)
 			if err != nil {
-				log.Warn("Rocksmq DestroyTopic failed!", zap.Any("topic", topic), zap.Any("error", err))
+				log.Warn("Failed to destroy topic in rocksmq!", zap.Any("topic", topic), zap.Any("error", err))
```
Comment (congqixia) [inline internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:197]: Maybe
``` zap.String("topic", topic), zap.Error(err)``` 
is better

**Pull request closed without merging**